docs: update readme remote (#202)
jina-ai/now: README.md

In a new conda environment first execute `conda install grpcio tokenizers protobuf`. Then run `pip install jina-now`.

### Usage
You can use the following command to start Jina NOW.
```bash
jina now start
```
First, you will get asked what search case you would like to deploy. 

### Quick Start
```bash
jina now start
```

> First you will be prompted to choose the modality. As for now, we support to search images or text. But in the future, we will add many more options here.

Reviewer: ```suggestion
First, you will be prompted to choose the modality. As for now, we support images or text searches. But in the future, we will add many more options here.
```

Likewise, if you chose docarray URL, you will be prompted to enter the URL.
In case you selected local path, `jina-now` will ask you to enter the local path of the data folder as shown bellow.

<img width="506" alt="question-local-path" src="https://user-images.githubusercontent.com/11627845/170256044-67e82e86-6439-4a3e-98f1-dbdf1940de67.png">

The search app can be deployed in different qualities. Have in mind that a better quality leads to a larger ai model being deployed and therefore inference will be a bit slower.

<img width="497" alt="question-quality" src="https://user-images.githubusercontent.com/11627845/170256049-18add461-f666-48f4-9dfe-52be9404a73d.png">

Currently, we provide two deployment options. We recommend using the cloud deployment. This will run your search app on our servers.
Alternatively, you can select the local deployment option.

<img width="547" alt="question-deployment" src="https://user-images.githubusercontent.com/11627845/170256038-8c44a5b8-985a-4fe7-af5d-16df0244f4bb.png">

In case of local deployment, you will be asked where you want to deploy it. Jina NOW reads your local .kube/config and lists all kubernetes clusters you have access to. 

> If you don't want to use an existing cluster, you can create a new one locally.
> <img width="643" alt="question-cluster" src="https://user-images.githubusercontent.com/11627845/170256027-99798fae-3ec4-42dc-8737-843f4a23f941.png">

Reviewer: ```suggestion
If you don't want to use an existing cluster, you can create a new one locally.

<img width="643" alt="question-cluster" src="https://user-images.githubusercontent.com/11627845/170256027-99798fae-3ec4-42dc-8737-843f4a23f941.png">
```

> After the program execution finished, two links will be shown to you. The first one brings you to a playground where you can run example queries and experiment with the search case.
> The second url leads you to the swagger ui which is useful for Frontend integration.

Reviewer: ```suggestion
After the program execution is finished, two links will be shown to you. The first one brings you to a playground where you can run example queries and experiment with the search case.
The second URL leads you to the swagger UI which is useful for Frontend integration.
```

<img width="709" alt="Screenshot 2022-05-26 at 16 34 56" src="https://user-images.githubusercontent.com/11627845/170511632-c741a418-1246-4c23-aadd-cfd74d783f6b.png">

Example of the playground.

<img width="350" alt="Screenshot 2022-05-26 at 16 36 49" src="https://user-images.githubusercontent.com/11627845/170511607-3fb810f7-a5aa-47cd-9f70-e6034a96b9fd.png">

Example of the swagger ui.

<img width="350" alt="Screenshot 2022-05-26 at 16 36 06" src="https://user-images.githubusercontent.com/11627845/170511580-230d1e41-5e14-4623-adb6-3d4b2d400dc9.png">

  
### Use CLI Parameters

> Instead of answering the questions manually, you can also send the parameters when starting Jina NOW like shown here.

Reviewer: ```suggestion
Instead of answering the questions manually, you can also provide command-line arguments when starting Jina NOW like shown here.
```
